Minor tweaks

Goooler opened the pull request.

renovate[bot] committed:
Add renovate.json

renovate[bot] committed:
chore(deps): update plugin org.ajoberstar.stutter to v0.7.2

Goooler committed:
Create gradle.properties

renovate[bot] committed:
Gradle 8.2

* chore(deps): update dependency gradle to v8.2

* Simple property assignments in KTS

---------

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Goooler <wangzongler@gmail.com>

Goooler committed:
KTS cleanups

renovate[bot] committed:
chore(deps): update dependency gradle to v8.2.1

TWiStErRob commented on build.gradle.kts:
```diff
@@ -32,20 +32,20 @@ dependencies {
 }
 
-tasks.withType<Test>() {
+tasks.test {
```
Original code included all the `compatTest...` tasks, not it's just "unit tests".

```suggestion
tasks.withType<Test>().configureEach {
```

Goooler replied:
Good catch!

Goooler committed:
Update build.gradle.kts

Co-authored-by: Róbert Papp <papp.robert.s@gmail.com>

Goooler committed:
Trigger CI by workflow_dispatch

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows

renovate[bot] committed:
chore(deps): update dependency gradle to v8.3

renovate[bot] committed:
chore(deps): update actions/checkout action to v4

renovate[bot] committed:
chore(deps): update dependency gradle to v8.4

TWiStErRob commented:
Mostly merged in https://github.com/ajoberstar/gradle-stutter/pull/34
